ovirtlago/prefix.py

```diff
@@ -49,6 +50,43 @@
 log_task = functools.partial(log_utils.log_task, logger=LOGGER)
 
 
+def _create_junitxml_fullfilename(
+    default_dir, default_filename, junitxml_file=None
```

Comment:
`junitxml_file` should be changed to be more generic

Reply:
changed filename to: output_filename